plugins/destination/plugin_testing_migrate.go

```diff
@@ -62,6 +65,9 @@ func testMigration(ctx context.Context, mem memory.Allocator, _ *testing.T, p *P
 	if err != nil {
 		return fmt.Errorf("failed to read all: %w", err)
 	}
+
+	sortRecordsBySyncTimeArrow(source, resourcesRead)
```

Comment:
This makes sure `Read()` returned our test rows in "insertion order"

Reply:
It's not necessary for it to be here if the destination's `Read()` always returns the results sorted, which is another option